docs/blog/20230903-confused-Techie-pulsars-ci.md

```diff
@@ -0,0 +1,141 @@
+- A Pulsar binary was built for every single platform we support, this means Windows, Linux, macOS, Apple Silicon, and ARM Linux.
+
+The big point of interest today is the process of building the Pulsar binaries.
+
+On August 17, 2022 [@mauricioszabo] [introduced](https://github.com/pulsar-edit/pulsar/commit/64427f41782ae4a2ab72a81762dc8f2bcb3f2f7e) and created Pulsar's integration with [CirrusCI](https://cirrus-ci.com/). Which is an overall, fantastic CI platform, that at the time had a very generous free tier. So generous in fact that every single binary built for Pulsar, since we started the project, has been built within this free tier.
+
+As those familiar with the project know, all of our funding comes from our fantastic community, via [OpenCollective](https://opencollective.com/pulsar-edit) and [GitHub Sponsors](https://github.com/sponsors/pulsar-edit), and we aim to be very careful about how we spend the money that is donated to us. At all times ensuring that the money is used in ways that can benefit the majority of our users, and that there is no better alternative for it's purpose. That's why when Cirrus [announced](https://cirrus-ci.org/blog/2023/07/17/limiting-free-usage-of-cirrus-ci/) that they were changing their free tier to something a bit more restrictive we had to scramble and find how we can continue our pace.
+
+## How it is, and how it got there
+
+Now, I don't want to come across as ungrateful to the fantastic service that Cirrus offers, so when this price change was first announced, we had originally intended to figure out if we could afford these price changes reasonably.
+
+To determine what we could afford, we first had to estimate what we would've been paying the past months of usage. [@DeeDeeG] was able to break down exactly what our costs looked like, using our previous months data.
+
+Keeping in mind that we trigger the same amount of runs per platform, the difference here is how slow or fast any given platform is during the build process:
+
+| Month | Windows Credits/Hours  | Linux Credits/Hours    | macOS Credits/Hours     |
+| ----- | ---------------------- | ---------------------- | ----------------------- |
+| May   | 66 credits / 273 hours | 72credits / 396 hours  | 490 credits / 544 hours |
+| July  | 55 credits / 229 hours | 39 credits / 213 hours | 305 credits / 339 hours |
+
+Using this data, where we consider 1 credits = $1 USD, minus the novel 50 free credits, July would've costed $349 USD. We could then realistically estimate that building Pulsar binaries would cost ~$350 USD a month.
+
+Which when we consider that's a higher price than anything the Pulsar team spends any money on for Pulsar, it's a bit of a tough pill to swallow. But especially because Cirrus isn't the only service out there, no one on the team could justify spending our sponsors money on such a large expense. At this point, we had to determine a way to lower our usage or move off the platform.
```

Comment:
```suggestion
Which when we consider that's a higher price than anything the Pulsar team spends any money on for Pulsar, it's a bit of a tough pill to swallow. But especially because Cirrus isn't the only service out there, no one on the team could justify spending our sponsor's money on such a large expense. At this point, we had to determine a way to lower our usage or move off the platform.
```